src/Dict.h

```diff
@@ -1057,10 +1057,11 @@ template <typename T> class Dictionary
 		{
 		ASSERT(table && -1 <= position && position < Capacity());
 
+		auto current_cap = Capacity();
```

Comment:
Wonder if for consistency with the other lines this should use `int` instead of `auto`, but not sure.

https://github.com/zeek/zeek/pull/2410/files#diff-38cf25ea2152135d0b4bbd2f78b1e6cd3357b3497f2fd9d53275ecabc2c3169dR1034

Reply:
May also apply for the other one.